styles/colors-silicon-day.less

```diff
@@ -185,14 +185,14 @@
 
 // Popup
 // ---------------------------------------
-@agate-popup-plain-body-color: @agate-accent;
-@agate-popup-bg-color: fade(black, 85%);
+@agate-popup-plain-body-color: @silicon-black;
+@agate-popup-bg-color: @silicon-white;
```

Comment:
If this is set instead to `@agate-background`, you could delete the corresponding line out of the night file. Same for all of the direct color assignments in this section:
```css
@agate-popup-plain-body-color: @agate-foreground;
@agate-popup-bg-color: @agate-background;
@agate-popup-border-color: @agate-foreground;
@agate-popup-body-bg-color: @agate-background;
```